tsdb/head_test.go

```diff
@@ -1875,3 +1875,66 @@ func TestHeadLabelNamesValuesWithMinMaxRange(t *testing.T) {
 		})
 	}
 }
+
+//func TestHeadCompactionRace(t *testing.T) {
```

Comment:
If you want to skip, just do `t.Skip`. Otherwise let's remove (:

Reply:
I will add `t.Skip` (didnt know about that, thanks!), dont want to remove as I will need it later